recipes/iteration_utilities/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "iteration_utilities" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "4e2964f27f5e2deb96232c7139a6d59aefd0eb130864546d98fce204959e834c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+  - vc9  # [win and py27]
+  - vc10  # [win and py34]
+  - vc14  # [win and py35]
+  skip: True  # [osx]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
Sorry I didn't realize this was compiling code before. Could we please add `toolchain`?